webpack.react.dev.js

```diff
@@ -44,6 +45,9 @@ module.exports = {
             filename: '[name].css',
             ignoreOrder: true,
```

Comment:
`ignoreOrder` is correct for compiled, as the compiled webpack plugins will automatically sort the CSS rules. If the css plugin is used outside of compiled, that may cause a potential issue.